docs/1-click-signup.mdx

```diff
@@ -999,5 +999,7 @@ This method requires additional work; mainly, you will have to build a UI for th
 
     This redirect will ultimately land the user back on your defined `redirectUrl` with no user interaction appearing as if only a page refresh has occurred.
 
+4. **If using `verificationOptions` `only_link` then the `/1-click` response body will not contain a `url` even if `match` is `true.** This is because there is no need to redirect the user. Thanks to the verification link sent via SMS already containing the code, there is no need for a redirect. Upon clicking the link, we will redirect the user to your defined `redirectUrl` upon phone verification and credential creation happening in a headless manner.
```

Comment:
```suggestion
4. **If you use `only_link` for `verificationOptions`, then the `/1-click` response body will not contain a `url` even if `match` is `true.** This is because user will receive a verification link by SMS, which means you don't need to handle redirecting them. When the user clicks the link, we will verify them and then redirect them to your defined `redirectUrl` in a headless manner.
```